src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```
так ці інтерфейси автоматично будуть переходити і всім наслідникам, де вже можна буде писати реалізацію їх методів